src/components/HtmlView.js

```diff
@@ -31,34 +29,32 @@ const cssRules =
   font-size: ${normalize(13)}px;
 }
 th {
-  border-bottom: 0.25px solid ${'#3f5c7a'};
-  border-left: 0.25px solid ${'#3f5c7a'};
-  border-top: 0.25px solid ${'#3f5c7a'};
+  border-bottom: 0.68px solid ${'#3f5c7a'};
+  border-left: 0.68px solid ${'#3f5c7a'};
+  border-top: 0.68px solid ${'#3f5c7a'};
 }
 th:last-child {
-  border-right: 0.25px solid ${'#3f5c7a'};
+  border-right: 0.68px solid ${'#3f5c7a'};
 }
 td {
-  border-bottom: 0.25px solid ${'#b5b5b5'};
-  border-left: 0.25px solid ${'#b5b5b5'};
+  border-bottom: 0.68px solid ${'#b5b5b5'};
+  border-left: 0.68px solid ${'#b5b5b5'};
   border-top: 0;
 }
 td:last-child {
-  border-right: 0.25px solid ${'#b5b5b5'};
+  border-right: 0.68px solid ${'#b5b5b5'};
 }
 table {
-  border-bottom: 0.25px solid ${'#b5b5b5'};
+  border-bottom: 0.68px solid ${'#b5b5b5'};
 }
 `;
 
-const renderers = {
-  table: makeTableRenderer({ WebView, cssRules })
-};
+const renderers = { table: TableRenderer };
 
 const htmlConfig = {
-  alterNode,
+  WebView,
```

Comment:
what is this `WebView` used for?

Reply:
`WebView` was previously used in `makeTableRenderer`. `makeTableRenderer` has been removed from the table plugin. `WebView` is used because of the table plugin.

https://github.com/native-html/plugins/tree/master/packages/table-plugin#readme